[2024-05-19T13:09:17.007+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-19T13:09:17.055+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_parse_store_report_weather_data.fetch_and_save_weather_data scheduled__2024-05-07T05:00:00+00:00 [queued]>
[2024-05-19T13:09:17.069+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_parse_store_report_weather_data.fetch_and_save_weather_data scheduled__2024-05-07T05:00:00+00:00 [queued]>
[2024-05-19T13:09:17.070+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2024-05-19T13:09:17.101+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): fetch_and_save_weather_data> on 2024-05-07 05:00:00+00:00
[2024-05-19T13:09:17.116+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2396) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-19T13:09:17.119+0000] {standard_task_runner.py:63} INFO - Started process 2414 to run task
[2024-05-19T13:09:17.122+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'fetch_parse_store_report_weather_data', 'fetch_and_save_weather_data', 'scheduled__2024-05-07T05:00:00+00:00', '--job-id', '5829', '--raw', '--subdir', 'DAGS_FOLDER/cris.py', '--cfg-path', '/tmp/tmpikw22jx4']
[2024-05-19T13:09:17.124+0000] {standard_task_runner.py:91} INFO - Job 5829: Subtask fetch_and_save_weather_data
[2024-05-19T13:09:17.237+0000] {task_command.py:426} INFO - Running <TaskInstance: fetch_parse_store_report_weather_data.fetch_and_save_weather_data scheduled__2024-05-07T05:00:00+00:00 [running]> on host d2eb9883ca8d
[2024-05-19T13:09:17.903+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_parse_store_report_weather_data' AIRFLOW_CTX_TASK_ID='fetch_and_save_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2024-05-07T05:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-07T05:00:00+00:00'
[2024-05-19T13:09:17.906+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-19T13:09:19.144+0000] {logging_mixin.py:188} INFO - Data has been successfully saved to the JSON file
[2024-05-19T13:09:19.145+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-19T13:09:19.146+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-19T13:09:19.159+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=fetch_parse_store_report_weather_data, task_id=fetch_and_save_weather_data, execution_date=20240507T050000, start_date=20240519T130917, end_date=20240519T130919
[2024-05-19T13:09:19.214+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-19T13:09:19.263+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-19T13:09:19.265+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
